requirements_for_test.txt

```diff
@@ -6,3 +6,4 @@ moto==2.0.6
 pytest-env==0.6.2
 pytest-mock==3.6.0
 pytest==6.2.3
+jinja2-cli
```

Comment:
should this be `jinja2-cli[yaml]`? (that's what we have in other test reqs in other repos)

Reply:
Good point - PyYaml is still installed elsewhere but we should be explicit here that it's needed.

I've amended the commit.